extensions/warp-ipfs/src/store/message.rs

```diff
@@ -1329,6 +1329,19 @@ impl MessageStore {
                         //TODO: Maybe add a api event to emit for when blocked users are added/removed from the document
                         //      but for now, we can leave this as a silent update since the block list would be for internal handling for now
                     }
+                    ConversationUpdateKind::ChangeSettings { settings } => {
+                        conversation.settings = settings;
```

Comment:
Shouldnt need to include the setting since its already apart of `conversation` in the update.

```suggestion
```